context/app/static/js/components/workspaces/AddDatasetsTable/hooks.ts

```diff
@@ -115,6 +107,20 @@ function useDatasetsAutocomplete({
     setAutocompleteValue(null);
   }, [setInputValue, setAutocompleteValue]);
 
+  const addDataset = useCallback(
+    (e: SyntheticEvent<Element, Event>, newValue: SearchAheadHit | null) => {
+      const uuid = newValue?._source?.uuid;
+
+      if (uuid) {
+        setAutocompleteValue(newValue);
+        updateDatasetsFormState([...selectedDatasets, uuid]);
+        resetAutocompleteState();
+        toastSuccess(`${newValue._source.hubmap_id} successfully added.`);
```

Comment:
Can you just remove the `setAutocompleteValue` instead?

Reply:
This still leaves the input value, but I can replace that call with `resetAutocompleteState()`